src/components/pages/order/navbar/NavbarRightSide.js

```diff
@@ -1,11 +1,41 @@
-import { Link } from "react-router-dom";
+
 import styled from "styled-components";
+import ToggleButton from "../../../reusable-ui/ToggleButton";
 import Profile from "./Profile";
+import { useState } from "react"
+import ToastAdmin from "./ToastAdmin";
+import { toast } from "react-toastify";
+
 
 export default function NavbarRigthSide({username}) {
+
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+
+  const displayToastNotification = () => {
+    if (!isModeAdmin) {
```

Comment:
```suggestion
    if (!isAdmin) {
```
( si renaming validé par Vi )